### ISSUE TITLE
archive courses

### ISSUE THREAD

**Issue by roelvisje** [chamilo/chamilo-lms#3528]
In a platform with many courses sometimes it could be a good idea to "clean up" some courses, but you still want to keep them to be able to restore them at a later time. so deleting them is out of the question. 

For this an "archive" could be of use where you can place your courses you are not using, but still want to keep for later.

In my perspective this could be at the "course list" page, where you have 2 tabs now (standard list and management list) . A 3rd tab could be "archive". Then from the standard/management list one could move the course to the archive. Of course from archive one can move it back to active course if needed.

**Comment by ywarnier** [comment]
Did you try the "hidden" visibility option for courses? What are you missing with that one?

**Comment by roelvisje** [comment]
yes, but they still end up in the long initial active list (as administrator ofcourse). I know that when you do advanced search you can show the hidden ones. i just figured it would be nicer to just have them seperate from the active courses and let them appear in a "hidden/archive" tab. so that is the main idea. but it is not that important ofcourse. it depends a bit on the workflow you want to have in the platform.

**Comment by ywarnier** [comment]
I suppose so. I'll set that as an enhancement but that's not really an urgent matter, so not setting a version yet.
